applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_fixed_temperature/test_thermal_fixed_temperature_2D3N_newmark/ProjectParameters.json

```diff
@@ -0,0 +1,142 @@
+{
+    "problem_data": {
+        "problem_name":         "test_thermal_fixed_temperature_2D3N",
+        "start_time":           0.0,
+        "end_time":             8640000,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "T",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "test_thermal_fixed_temperature_2D3N"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "MaterialParameters.json"
+        },
+        "time_stepping":              {
+            "time_step":                 600,
+            "max_delta_time_factor":    100
```

Comment:
This test is a copy of a backward euler version. To get it to work, we needed to reduce this max_delta_time_factor from 1000 to 100, apparently Newmark does not tolerate very large time steps.